app/views/mechanics/show.html.erb

```diff
@@ -0,0 +1,4 @@
+<h1>Mechanic Show Page </h1>
+<p> Name: <%= @mechanic.name %></p>
+<p> Years experience: <%= @mechanic.years_experience %></p>
+<p> Rides being wokred on: 
```

Comment:
It is in here that you will iterate over your `@mechanic.rides` to pull the rides associated with that mechanic. 
```ruby
<% @mechanic.rides.each do |ride| %>
  <ul id="mechanic_rides-<%= ride.id %>"> #this is here because I used a within block for testing
    <li><%= ride.name %></li>
  </ul>
<% end %>
```